presto-main/src/test/java/io/prestosql/operator/TestDynamicFilterSourceOperator.java

```diff
@@ -255,6 +266,77 @@ public void testCollectWithRealNaN()
                         new DynamicFilterId("0"), Domain.multipleValues(REAL, ImmutableList.of((long) floatToRawIntBits(42.0f)))))));
     }
 
+    @Test
+    public void testCollectTooMuchRowsDouble()
```

Comment:
could we have a parametrized test method that we call with different input blocks, input types and expected result tuple domain?

Reply:
done